src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -115,9 +115,6 @@ public static bool IsAttributeNameContext(this SyntaxTree syntaxTree, int positi
             //   int i;
             //   |
 
-            // class C {
```

Comment:
Do we need to actually handle this case instead of removing the comment?

Reply:
The second overload below handles cases with modifiers, and depends on this overload not handling them. I see similar path for `IsTypeDeclarationContext` overloads. Either they're all wrong or for some reason by design. Before we have answer, let's make the comment acurate so not to cause misleading further.

Reply:
So I took a closer look at the code here, and you are right, there's definitely a pattern. However, what I couldn't figure out is why we have those overloads and when should I choose one over the other. I'm just really curious, maybe @jasonmalinowski can provide some context here?

That said, remove the comment is fine, and we might as well remove the one in `IsTypeDeclarationContext` as well.

Reply:
OK, I'll remove that comment, too.